ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Reporter.scala

```diff
@@ -102,14 +102,19 @@ object Reporter {
           }
       }
 
-    override def report(results: Vector[LedgerTestSummary]): Unit = {
+    override def report(results: Vector[LedgerTestSummary], testsSeed: String): Unit = {
       s.println()
       s.println(blue("#" * 80))
       s.println(blue("#"))
       s.println(blue("# TEST REPORT"))
       s.println(blue("#"))
       s.println(blue("#" * 80))
 
+      s.println()
+      s.println(yellow("### CONFIG"))
```

Comment:
This is not really configuration (it can't be configured). Probably "RUN INFORMATION" makes more sense.

Reply:
fixed